README.md

```diff
@@ -462,6 +464,16 @@ and every `send_mail` call will appear instantly in the UI.
 > EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 > ```
 
+### Installing The Required LaTeX Packages For Local Development
+
+In order to generate the equations in Wagtail using LaTeX strings via Matplotlib (Non-JS Equations), we will need to use `MacPorts` to install the following packages `texlive-latex-extra`.
+
+As a prerequisite you will have to have [MacPorts](https://www.macports.org/install.php) installed.
+
+```bash
+sudo port install texlive-latex-extra
+```
```

Comment:
Just a heads up, although installing the .pkg file is supposed to automatically update your PATH so that MacPorts can be found, this didn't happen for me (prob some local issue). I had to manually add it myself. So, if devs encounter issues with MacPorts not being recognised, they should add the following to their PATH.

```
export PATH=/opt/local/bin:/opt/local/sbin:$PATH
```

The documentation suggests that the installer should add it automatically when using the pkg installer https://guide.macports.org/#installing.shell.postflight.